src/integration-test/java/uk/ac/ebi/tsi/portal/api/team/TeamRestControllerIT.java

```diff
@@ -136,34 +158,102 @@ public void getAllTeamsForCurrentUser() throws Exception {
 		)
 				.andExpect(status().isOk());
 	}
-		/*@Test
-		public void add_member_to_team() throws Exception{
-			
-			String json = "{\"name\":\"" + teamName + "\", \"memberAccountEmails\":[\""+ email + "\"]}";
-			logger.info("In add member to team " + json);
-			mockMvc.perform(
-					post("/team/member") 
-					.headers(createHeaders(token))
-					.contentType(MediaType.APPLICATION_JSON)
-					.content(json)
-					.accept(MediaType.APPLICATION_JSON))
 
-			.andExpect(status().isOk());
+	@Test
+	@WithMockUser(username = "usr-e8c1d6d5-6bf4-4636-a70e-41b8f32c70b4")
+	public void team_owner_can_see_team_manager_emails() throws Exception {
 
-		}
-		
+		String domainManagersUrl = "/domains/dom-e0de1881-d284-401a-935e-8979b328b158/managers";
+		String managersString = mapper.writeValueAsString(getManagers());
+		mockDomainService.givenThat(WireMock.get(domainManagersUrl).willReturn(okJson(managersString)));
+		mockMvc.perform(
+				get("/team/test-team1")
+						.header("Authorization", "Bearer sometoken")
+						.contentType(MediaType.APPLICATION_JSON)
+						.accept(MediaType.APPLICATION_JSON))
+				.andExpect(status().isOk())
+				.andExpect(jsonPath("$.managerEmails", hasSize(1)));
+	}
 
-		@Test
-		public void canRemoveMemberFromTeam() throws Exception{
+	@Test
+	@WithMockUser(username = "usr-b070585b-a340-4a98-aff1-f3de48da8c38")
+	public void non_team_owner_can_see_team_manager_emails() throws Exception {
```

Comment:
I am not sure, what is going on in the test.
1) Everybody should be able to retrieve some basic information about the team.
2) The name of test is misleading (are you checking that 'non_team_owner_can_see_team_manager_emails' - is that allowed? It should not be)
3) What is that assert in catch phrase actually doing? What kind of error are you expecting.

Reply:
Refactored name, non-team owners cannot see. The exception is not thrown here, removed it.